integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/InfinispanStoreJdbcIT.java

```diff
@@ -32,7 +34,10 @@
 
    @Deployment
    public static Archive<?> deployment() {
-      return ShrinkWrap.create(WebArchive.class, "jdbc.war").addClass(InfinispanStoreJdbcIT.class).add(manifest(), "META-INF/MANIFEST.MF");
+      return ShrinkWrap.create(WebArchive.class, "jdbc.war")
+            .addClass(InfinispanStoreJdbcIT.class)
+            .addAsResource("jdbc-config.xml")
+            .add(manifest(), "META-INF/MANIFEST.MF");
```

Comment:
yes that's https://issues.jboss.org/browse/ISPN-7572

In addition, wondering here if for other modules (the ones which we don't know about and hence will not be able to list in org.infinispan:main) should have a better error to point out that this dependency is needed .. ?